test/verify/check-system-journal

```diff
@@ -329,6 +329,31 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
 
         self.allow_journal_messages(".*Data from the specified boot .* is not available: No such boot ID in journal.*")
 
+    def testIdentifierReloading(self):
+        b = self.browser
+        m = self.machine
+        tag_error = "check-error-inside"
+        tag_info = "check-info-inside"
+        m.execute(f"logger -p user.err --tag {tag_error} {tag_error.upper()}")
+        m.execute(f"logger -p 5 --tag {tag_info} {tag_info.upper()}")
+
+        self.login_and_go("/system/logs")
+        self.browser.select_PF4("#journal-prio-menu", "Info and above")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_error.upper()}')")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_info.upper()}')")
+
+        self.browser.select_PF4("#journal-prio-menu", "Error and above")
+        b.reload()
+        b.enter_page("/system/logs")
+        b.select_PF4("#journal-identifier-menu .pf-v5-c-select__toggle", tag_error)
+        b.wait_not_present(f"#journal-identifier-menu + ul button:contains('{tag_info}')")
+
+        self.browser.select_PF4("#journal-prio-menu", "Debug and above")
+        b.select_PF4("#journal-identifier-menu .pf-v5-c-select__toggle", tag_error)
```

Comment:
Does this reset once you change to "Debug and above"?

Reply:
Sure, there is more places where this issue could happen until src code fix, maybe the fix could be to call JS that does this renewing of identifiers, after each action and reload the identifiers instead of reloading page.
This page reloading is expensive so I wanted to avoid to call it more times.

Reply:
aha, you verify it is present by changing to it

Reply:
exactly, it is more precise, to also prove it is selectable, not just some text somewhere.